<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>agregar_usur_Foto</class>
 <widget class="QDialog" name="agregar_usur_Foto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>625</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Asistente para agregar nuevo usuario</string>
  </property>
  <widget class="QLabel" name="lbl_ImageDisplay">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>69</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Eliminar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>490</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>&amp;Eliminar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_TomarFoto">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>570</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>&amp;Tomar Foto</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Atras">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>580</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Atr√°s</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Continuar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>580</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Continuar</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comBox_CamarasList">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>189</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>65</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Camara:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>40</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Capturas</string>
   </property>
  </widget>
  <widget class="QListView" name="lview_Capturas">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>70</y>
     <width>131</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
